app/scripts/filters/resources.js

```diff
@@ -1012,6 +1012,20 @@ angular.module('openshiftConsole')
       return 'PAUSED_PENDING_INPUT' === stage.status;
     };
   })
+  .filter('deploymentStrategyNameToParams', function () {
```

Comment:
I'd rather the filter give me the strategy params instead of the property name.

```js
.filter('strategyParams', function() {
  return function(deploymentConfig) {
    var strategy = _.get(deploymentConfig, 'spec.strategy.type');
    switch (strategy) {
    case 'Recreate':
      return _.get(deploymentConfig, 'spec.strategy.recreateParams', {});
    // ...
    default:
      return null;  
    }
  };
});
```

Then in the directive `link` function, you can get the strategy params with

```js
$scope.$watch('deploymentConfig', function(deploymentConfig) {
  $scope.strategyParams = $filter('strategyParams')(deploymentConfig);
});
```

(Should have made that comment on first review, sorry.)